### DISCUSSION TITLE
Failed to render latex

### DISCUSSION THREAD

**Discussion by vinoddotcom** [KaTeX/KaTeX#3827] [Show and tell]
Encountered an issue while attempting to render a LaTeX expression using Katex, a library commonly used for mathematical typesetting. The LaTeX expression is as follows:

```
F(x) = \left\{ {\begin{array}{*{20}{c}} {0;x < - 1}\\ {\frac{1}{2}{{(x + 1)}^2}; - 1 \le x < 0}\\ {1 - \frac{{{{(1 - x)}^2}}}{2};0 \le x < 1}\\ {1.1 < x < \infty } \end{array}} \right.
``` 

However, when attempting to render this expression with Katex, I encountered the following error message: "KaTeX parse error: Unknown column alignment: * at position 31: …{\begin{array}{*̲{20}{c}} {0;x <…".

Interestingly, I tested the same LaTeX expression using `MathJax`, another popular library for rendering mathematical equations on the web, and it rendered correctly without any errors.

To verify the validity of the LaTeX expression, I used an online LaTeX renderer called "arachnoid," which employs multiple LaTeX rendering libraries. All four rendering libraries in [arachnoid](https://arachnoid.com/latex/) successfully rendered the expression without any issues.

I believe there might be a compatibility or syntax issue specific to Katex when dealing with complex LaTeX expressions like the one mentioned above. I would appreciate any assistance or insights into resolving this problem.

**Comment by vinoddotcom** [comment]
When we replace `*{20}{c}` with `{ll}` its work fine

**Comment by edemaine** [comment]
Yes, this is known issue #269 (no support for `*` in array environments).
